src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -114,23 +114,15 @@ def _generate_parameter_files(
     for key in ens_config.parameters:
         node = ens_config[key]
 
-        if isinstance(node, GenKwConfig):
-            gen_kw_dict = node.save(Path(run_path), iens, fs)
-            exports.update(gen_kw_dict)
-            continue
-        if isinstance(node, ExtParamConfig):
-            node.save(Path(run_path), iens, fs)
-            continue
-        if isinstance(node, ParameterConfig):
-            # For the first iteration we do not write the parameter
-            # to run path, as we expect to read if after the forward
-            # model has completed.
-            if node.forward_init and iteration == 0:
-                continue
-            node.save(Path(run_path), iens, fs)
+        # For the first iteration we do not write the parameter
+        # to run path, as we expect to read if after the forward
+        # model has completed.
+        if node.forward_init and iteration == 0:
```

Comment:
I guess its safe to assume all possible `node` here are subclass of `parameter_config` ?